app/src/test/java/com/anysoftkeyboard/keyboards/views/AnyKeyboardViewBaseTest.java

```diff
@@ -152,6 +153,18 @@ public void testWithLongPressOutputRegularPressKeyPressState() {
         Assert.assertArrayEquals(provider.KEY_STATE_NORMAL, key.getCurrentDrawableState(provider));
     }
 
+    @Test
```

Comment:
Also add a test that verify that long press is repeated.
I think that's possible using `ViewTestUtils.navigateFromTo